src/test/scala/uk/gov/hmrc/perftests/example/TrackingConsentFrontendRequests.scala

```diff
@@ -40,8 +40,20 @@ object TrackingConsentFrontendRequests extends ServicesConfiguration {
       .check(status.is(200))
   }
 
+  val getLiveHmrcFrontendVersion: String = {
```

Comment:
I guess this could be `getStagingHmrcFrontendVersion` if that's less confusing?

Reply:
maybe `currentHmrcFrontendVersion` - or just `hmrcFrontendVersion` ?